src/entity_extraction/training/spacy_ner/create_config.py

```diff
@@ -0,0 +1,40 @@
+# Author: Jenit Jain
+# Date: 2023-06-21
+# Inspired from https://github.com/explosion/projects/blob/v3/pipelines/ner_demo_update/scripts/create_config.py
+"""This script create a config file when resuming training of a spacy model from a past checkpoint
+
+Usage: create_config.py --model_path=<model_path> --output_path=<output_path>
+
+Options:
+    --model_path=<model_path>         The path to the model artifacts.
+    --output_path=<output_path>       The path to the output config file.
+"""
+
+from pathlib import Path
+from docopt import docopt
+import spacy
+
+opt = docopt(__doc__)
```

Comment:
bug: this should be in the `main` function because if any other script tries to use any code from this it will try to read the mandatory arguments and say they're not found.